koku/masu/api/additional_context.py

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright 2023 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""View for modifying the additional context field of a schema."""
+from django.views.decorators.cache import never_cache
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.decorators import permission_classes
+from rest_framework.decorators import renderer_classes
+from rest_framework.permissions import AllowAny
+from rest_framework.response import Response
+from rest_framework.settings import api_settings
+
+from masu.database.provider_db_accessor import ProviderDBAccessor
+
+ADDITIONAL_CONTEXT_KEYS = ["aws_list_account_aliases", "crawl_hierarchy"]
+
+
+def opt_dict_serializer(op_dict):
+    """Checks that the opt dict is structure correctly."""
+    if not isinstance(op_dict, dict):
+        return "Post body must be a list of dictionaries."
+    for rk in ["op", "key"]:
+        if not op_dict.get(rk):
+            return f"Missing key in body ({rk})."
+    if op_dict.get("key") not in ADDITIONAL_CONTEXT_KEYS:
+        return f"Invalid key supplied: {op_dict.get('key')}"
+    if op_dict.get("op") not in ["remove", "replace"]:
+        return f"Unrecognized op: {op_dict.get('op')}"
```

Comment:
Might be better to say "Only the `remove` and `replace` operations are supported"